src/main/resources/META-INF/maven/site-macros.vm

```diff
@@ -678,11 +682,11 @@ $indent     </ul>##
 #*  *##if ( $sideBarEnabled )
 #*    *##set( $dataSize = "medium" )
 #*    *##set( $dataAlign = "left" )
-    <div id="twitter">
+    <div id="twitter" style="border:none; margin-top: 10px">
 #*  *##else
 #*    *##set( $dataSize = "large" )
 #*    *##set( $dataAlign = "right" )
-    <ul class="nav pull-right"><li>
+    <ul class="nav pull-right" style="border:none; margin-top: 7px"><li>
```

Comment:
Why did you modify these?

Reply:
Because Twitter button was start from top without margin. 
Google search and facebook has some margin from top.
It didn't look good.
You can see original. eg: https://maven.apache.org/skins/maven-fluido-skin/mskins-24_topbar/index.html

Reply:
before:
![image](https://user-images.githubusercontent.com/3578921/77567288-3bdb5a80-6ec7-11ea-9e9f-f6289178325d.png)

after:
![image](https://user-images.githubusercontent.com/3578921/77567351-50b7ee00-6ec7-11ea-9bd3-f6f6348841cf.png)